plugin/al/usdtransaction/AL/usd/transaction/tests/CMakeLists.txt

```diff
@@ -41,17 +43,55 @@ add_test(
     ${USDTRANSACTION_TEST_EXECUTABLE_NAME}
 )
 
+if(IS_WINDOWS)
+  set(path
+      "${CMAKE_INSTALL_PREFIX}/lib/gtest"
+      "${CMAKE_INSTALL_PREFIX}/plugin/al/lib"
+      "$ENV{PATH}"
+  )
+  string(REPLACE ";" "\;" path "${path}")
+
+  set_property(TEST ${USDTRANSACTION_TEST_NAME} APPEND PROPERTY ENVIRONMENT 
+      "PATH=${path}"
+  )
+endif()
+
 add_test(
   NAME ${USDTRANSACTION_PYTHON_TEST_NAME}
   COMMAND
     python -m unittest discover -s ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
I missed this. We need to use Maya python for running the test.

Reply:
This still shows up as "python" in my review...?

Reply:
Fixed in 730580975

Reply:
this still seems to be `python -m ...` instead of `${MAYA_PY_EXECUTABLE} -m ...`, even in 7305809

Reply:
Ah my bad! missed to add the change for this in this commit. Will have the fix in the next commit,

Reply:
Cool, see it in b11cb1de!